scrapyrt/core.py

```diff
@@ -164,7 +164,7 @@ def _get_log_file_path(self):
         log_dir = os.path.join(self.log_dir, self.spider_name)
         if not os.path.exists(log_dir):
             os.makedirs(log_dir)
-        filename = datetime.datetime.now().isoformat() + '.log'
+        filename = datetime.datetime.now().strftime('%Y-%m-%dT%H;%M;%S.%f') + '.log'
```

Comment:
so this produces string like this 

```
2016-12-08T11;31;06.372041
```
this is not valid date format. Perhaps slash might be better here? e.g.

```
2016-12-08T11-31-06-372041
```

slash as separator seems more readable to my eyes
?